packages/api/check.js

```diff
@@ -0,0 +1 @@
+module.exports = require("./dist/src/check")
```

Comment:
Should JS and .D.TS files be on github tho? 🤔

Reply:
Great question!

I would actually like to have it as a *.ts file, but that would mean we would need to compile those files _in place_ (not to the `dist` directory), because I want this import to work: `import { File } from '@stryker-mutator/api/core'` and that can only be done with a file called "core.js" inside api (on node < 12.7).

When we drop support for node < 12.7 we can start using [`exports`](https://nodejs.org/dist/latest-v14.x/docs/api/packages.html#packages_exports) and remove these root files altogether.

I don't want to make an exception for some files to be compiled in place, because that also means making an exception in our clean script and making the output files hidden in vscode config (i don't like hidden files 😢)

Do you agree with this? In short: we should be able to remove these files once we can use `exports` in package.json.